src/dotnet/UI.Blazor.App/Services/audio-context-source.ts

```diff
@@ -180,7 +181,9 @@ abstract class AudioContextSourceBase implements AudioContextSource {
         // It might be in suspended state in this case.
         const context: OverridenAudioContext = new AudioContext({
             latencyHint: 'balanced',
-            sampleRate: this.purpose === 'playback' ? AP.SAMPLE_RATE : AR.SAMPLE_RATE,
+            sampleRate: this.purpose === 'playback'
```

Comment:
do you really need two ternary operators here?

Reply:
I don't have good ideas, how to get rid of them